src/components/TodoItem.tsx

```diff
@@ -0,0 +1,154 @@
+import classNames from 'classnames';
+import { useState } from 'react';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  todo: Todo | null;
+  loadingTodoIds: number[];
+  deleteClickHandler: (
+    id: number
+  ) => void;
+  patchHandlerTodoCompleted: (
+    id: number,
+    completed: boolean,
+    bringInList: boolean,
+  ) => void;
+  patchHandlerTodoTitle:(
+    id: number,
+    title: string
+  ) => void;
+  setLoaderTodo: (loaderTodo: number) => void;
+  loaderTodo: number;
+};
+
+export const TodoItem: React.FC<Props> = ({
+  todo,
+  loadingTodoIds,
+  deleteClickHandler,
+  patchHandlerTodoCompleted,
+  patchHandlerTodoTitle,
+  setLoaderTodo,
+  loaderTodo,
+}) => {
+  const { id, title, completed } = todo || { id: 0 };
+  const [clickedForm, setClickedForm] = useState(false);
+  const [titleText, setTitleText] = useState(title || '');
+
+  const changeValueHandler = (event: React.FormEvent<HTMLInputElement>) => {
```

Comment:
```suggestion
  const handleChangeValue = (event: React.FormEvent<HTMLInputElement>) => {
```

Reply:
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364

Fix it everywhere to be consistent with naming convention

Reply:
Thanks for the article. Very informative!